components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/request/PostAuthnMissingChallengeQuestionsHandler.java

```diff
@@ -91,12 +91,18 @@ public PostAuthnHandlerFlowStatus handle(HttpServletRequest httpServletRequest,
                                              HttpServletResponse httpServletResponse,
                                              AuthenticationContext authenticationContext)
             throws PostAuthenticationFailedException {
+        String forceChallengeQuestionConfig = "";
 
         if (log.isDebugEnabled()) {
             log.debug("Post authentication handling for missing security questions has started");
         }
-        String forceChallengeQuestionConfig = getResidentIdpProperty(getAuthenticatedUser(authenticationContext).getTenantDomain(),
-                IdentityRecoveryConstants.ConnectorConfig.FORCE_ADD_PW_RECOVERY_QUESTION);
+        try {
+            forceChallengeQuestionConfig = getResidentIdpProperty(getAuthenticatedUser(authenticationContext).
+                    getTenantDomain(), IdentityRecoveryConstants.ConnectorConfig.FORCE_ADD_PW_RECOVERY_QUESTION);
+        } catch (Exception e) {
+            log.error("Error occurred while extracting resident IDP property for Force Missing Challenge Question " +
```

Comment:
shall we reword as 'Error occurred while retrieving resident IDP property for forced challenge questions' ?